### DISCUSSION TITLE
Disable Uptime Monitor?

### DISCUSSION THREAD

**Discussion by RainrZufall** [tobychui/zoraxy#355] [Q&A] [answered]
Hi, I'm currently testing zoraxy as an alternative to NPM.

So far it's working well, but the uptime monitor is causing me problems.
It prevents my NAS from going into standby.

Is there a way to deactivate the uptime monitor for individual domains or completely?

**Comment by tobychui** [comment, marked as answer]
Oh that is a good point, No, currently there is no uptime manager disable function. But I think this might be a valid enhancement to consider. Please fill out an enhancement issue form if you really want this feature to be added.

**Comment by overcuriousity** [comment]
I agree with the OP and submitted an [issue](https://github.com/tobychui/zoraxy/issues/360).
